test/adagrams.test.js

```diff
@@ -120,7 +154,10 @@ describe("Adagrams", () => {
     });
 
     it("returns a score of 0 if given an empty input", () => {
-      throw "Complete test";
+      //throw "Complete test";
+      [null, '', undefined].forEach(input => {
+        expect(scoreWord(input)).toBe(0);
```

Comment:
Very robust! Note: `undefined` is a common value to propagate throughout code in case of an error in JS. You may want to raise an error in a case like this so it doesn't just silently propagate further (returning `0` might give the assumption of just an input being empty. What if we are expecting to work with a string? We could leave it JS to catch, but who knows when that will happen in our code.)